src/components/ModelizerDrawView.vue

```diff
@@ -143,6 +161,7 @@ async function dropHandler(event) {
       data.plugin,
       `${defaultFolder.value}${route.query.path}`,
       componentDefinition,
+      componentPosition,
```

Comment:
`new ComponentDrawOption(componentPosition)`
 to avoid loosing `ComponentDrawOption` class reference.